examples/hello-world-udp-proxy/dvc-proxy-config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "instances": [
+    {
+      "sdkKey": "${DEVCYCLE_SDK_KEY}",
+      "unixSocketPath": "/tmp/dvc_lb_proxy.sock",
+      "unixSocketEnabled": true,
+      "unixSocketPermissions": 777,
```

Comment:
```suggestion
      "unixSocketPermissions": "0777",
```

Reply:
I forgot octal permissions had were a specific thing with the "0" in front 🤦🏻

Reply:
yep - gotta love filemodes!